addon/components/pixi-canvas.js

```diff
@@ -7,19 +7,41 @@ const {
 } = Ember;
 
 export default Component.extend({
-  pixiRenderer: computed('width', 'height', function() {
+
+  height: computed('height', {
+    get() { return 600; },
```

Comment:
Why 600? This would have to return the actual value.

Reply:
I thought i return 600 since its the default.
Once height get set this will be overwritten?

Reply:
There don't seem to be defaults in the code in `master` though. Also as this is a settable property this will never be overridden but always return 600 regardless of which value you might have set.

Reply:
Its the default in the PixiJS API Documentation.

It does not always return 600 -> [Twiddle](https://ember-twiddle.com/445e296f275b557453f6aa5aea7eff9c?openFiles=controllers.application.js%2C)

Reply:
> Its the default in the PixiJS API Documentation.

Reply:
> It does not always return 600 -> Twiddle

If you force recalculation of the CP in between calls (e.g. by adding `.volatile()`) you see it returns 600 again. If I'm not completely missing something the way to do writable CPs in Ember is

```js
height: computed({
  get() {
    return this._height;
  },
  set(key, value) {
    this._height = value;
    this.resizePixiRenderer();
    return value;
  }
}),
```

Reply:
This would be the same wouldn't it?
I think to introduce a new private property is not needed in this case.

```javascript
  height: computed({
    get() { },
    set(key, value) {
      this.resizePixiRenderer();
      return value;
    }
  }),
```

To use volatile the component would need to be reopened or extended by another component and then the height/width must be overwritten anyway to make them volatile.

Reply:
> To use volatile the component would need to be reopened or extended by another component and then the height/width must be overwritten anyway to make them volatile.

My point is I believe this works by accident, not by design.

Reply:
Its used in the [Ember PromiseProxy Mixin](https://github.com/emberjs/ember.js/blob/v2.0.1/packages/ember-runtime/lib/mixins/promise_proxy.js#L138) so i think its by design.

Reply:
You're right - good to know 👍

Reply:
:)
It would be nice if you could take a look at the tests, I dont have time now. Otherwise it will take some days until i can take a look at it.

Reply:
> Otherwise it will take some days until i can take a look at it.

Sure, no rush - ping me when this is ready